src/pages/panel/views/main.js

```diff
@@ -287,83 +329,103 @@ export default {
             </panel-actions>
           `}
         `}
-        ${options.terms
-          ? store.ready(stats) &&
-            !managedConfig.disableUserControl &&
-            html`
-              <panel-pause
-                onaction="${globalPause ? revokeGlobalPause : togglePause}"
-                paused="${paused || globalPause}"
-                global="${globalPause}"
-                managed="${paused?.managed}"
-                assist="${paused?.assist}"
-                revokeAt="${globalPause?.revokeAt || paused?.revokeAt}"
-                data-qa="component:pause"
+        ${!options.terms &&
```

Comment:
code here becomes quite long, maybe its worth to encapsulate it in own component?